dataland-frontend/src/utils/Email.ts

```diff
@@ -0,0 +1,17 @@
+import { type Card } from "@/types/ContentTypes";
+
```

Comment:
Side note:
The following files seem not to be needed/used/required anymore:

`ViewTeaserCompanyData.vue`; and if that is removed => `JoinDatalandButton.vue`

Reply:
Not sure about it, @fsoffel ?

Reply:
It is still reachible via its route.